api/v1beta1/disruption_types.go

```diff
@@ -420,16 +420,26 @@ func (status *DisruptionStatus) RemoveTargets(toRemoveTargetsCount int) {
 	}
 }
 
-var NonReinjectableDisruptions = []chaostypes.DisruptionKindName{
-	chaostypes.DisruptionKindGRPCDisruption,
+var NonReinjectableDisruptions = map[chaostypes.DisruptionKindName]interface{}{
```

Comment:
You may prefer an empty struct `map[chaostypes.DisruptionKindName]struct{}{` and value being `{}` this hold nothing in memory while an interface hold either the pointer or am I nil.